types/iopipe/index.d.ts

```diff
@@ -0,0 +1,37 @@
+// Type definitions for iopipe 1.6
+// Project: https://github.com/iopipe/the-meta-package#readme
+// Definitions by: Javon Harper <https://github.com/javonharper>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface LibraryConfig {
+    debug?: boolean;
+    token?: string;
+    networkTimeout?: number;
+    timeoutWindow?: number;
+}
+
+type FunctionWrapper = (wrappedFunction: any) => void;
+
+declare module "@iopipe/iopipe" {
```

Comment:
Instead of using ambient module declarations, these should be in separate packages: `types/iopipe__iopipe` and `types/iopipe__core`

[See more in the README](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d5d4960aae785017bbeb9afff5860f3271895eda/README.md#what-about-scoped-packages) on scoped packages.

Reply:
Gotcha, thanks!